arcgis-ios-sdk-samples/Utility Network/Perform valve isolation trace/PerformValveIsolationTrace.storyboard

```diff
@@ -0,0 +1,116 @@
+                                                    <rect key="frame" x="0.0" y="-17" width="70" height="34"/>
+                                                    <constraints>
+                                                        <constraint firstAttribute="width" relation="greaterThanOrEqual" constant="70" id="Yay-jv-ubm"/>
+                                                    </constraints>
+                                                    <fontDescription key="fontDescription" type="system" pointSize="14"/>
+                                                    <nil key="textColor"/>
+                                                    <nil key="highlightedColor"/>
+                                                </label>
+                                                <switch opaque="NO" contentMode="scaleToFill" horizontalHuggingPriority="750" verticalHuggingPriority="750" contentHorizontalAlignment="center" contentVerticalAlignment="center" on="YES" translatesAutoresizingMaskIntoConstraints="NO" id="o40-Kg-nZ5">
+                                                    <rect key="frame" x="75" y="0.0" width="51" height="0.0"/>
+                                                </switch>
+                                            </subviews>
+                                            <color key="backgroundColor" white="0.0" alpha="0.0" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
+                                            <constraints>
+                                                <constraint firstItem="o40-Kg-nZ5" firstAttribute="top" relation="greaterThanOrEqual" secondItem="5Ke-ZQ-uR3" secondAttribute="top" id="AwD-XY-htV"/>
+                                                <constraint firstItem="o40-Kg-nZ5" firstAttribute="centerY" secondItem="5Ke-ZQ-uR3" secondAttribute="centerY" id="CKa-zA-2j6"/>
+                                                <constraint firstItem="o40-Kg-nZ5" firstAttribute="leading" secondItem="lHG-2p-x4N" secondAttribute="trailing" constant="5" id="NFe-fK-sZR"/>
+                                                <constraint firstAttribute="bottom" relation="greaterThanOrEqual" secondItem="o40-Kg-nZ5" secondAttribute="bottom" id="dSk-3Z-IrV"/>
+                                                <constraint firstItem="lHG-2p-x4N" firstAttribute="centerY" secondItem="5Ke-ZQ-uR3" secondAttribute="centerY" id="hNs-WR-ext"/>
+                                                <constraint firstItem="lHG-2p-x4N" firstAttribute="leading" secondItem="5Ke-ZQ-uR3" secondAttribute="leading" id="odc-r9-WBt"/>
+                                                <constraint firstAttribute="width" relation="greaterThanOrEqual" constant="130" id="ztT-pV-wjE"/>
```

Comment:
The view shouldn't need a width constraint. If you have an unbroken line of constraints from the leading to the trailing edge, then the view should be self-sizing.